From 9e57da41bc40346a7e4d48a69484bdb63dfb86e4 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Sat, 15 Jul 2023 10:46:42 +0200
Subject: [PATCH 24/55] busybox: enable the watchdog tool

procd can currently only handle the main wdt. Enable the busybox
applet until procd can handle multiple devices.

Signed-off-by: John Crispin <john@phrozen.org>
---
 package/utils/busybox/Config-defaults.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/utils/busybox/Config-defaults.in b/package/utils/busybox/Config-defaults.in
index f88f395e2c..7812fc2bf5 100644
--- a/package/utils/busybox/Config-defaults.in
+++ b/package/utils/busybox/Config-defaults.in
@@ -2236,7 +2236,7 @@ config BUSYBOX_DEFAULT_WATCHDOG
 	default n
 config BUSYBOX_DEFAULT_FEATURE_WATCHDOG_OPEN_TWICE
 	bool
-	default n
+	default y
 config BUSYBOX_DEFAULT_FEATURE_IPV6
 	bool
 	default y if IPV6
-- 
2.34.1

